projectceladon/device-androidia
# Summary: Enable pstore feature on cel-apl

Description:
- use 'ram_dummy' backend to enable pstore
- specify mem_address=0x50000000 and mem_size=0x400000 for
  pstore saving kernel panic log.
Jira: https://jira01.devtools.intel.com/browse/OAM-67221
Test:
- use 'echo c > /proc/sysrq-trigger' to trigger a kernel panic fake
- check /data/logs/history_event file, there is a IPANIC_FAKE event in it.
Signed-off-by: Duan, YayongX <yayongx.duan@intel.com>
